packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -336,7 +352,8 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, ClrDa
     // setting the private variable to not retrigger the setter logic
     this._sortOrder = this._sort.reverse ? ClrDatagridSortOrder.DESC : ClrDatagridSortOrder.ASC;
     // Sets the correct icon for current sort order
-    this.sortIcon = this._sortOrder === ClrDatagridSortOrder.DESC ? 'arrow down' : 'arrow';
+    this.sortDirection = this._sortOrder === ClrDatagridSortOrder.DESC ? 'down' : 'up';
```

Comment:
This looks ok to me. The api for calling this function is the same. the sort fn is using `this.sortDirection` internally and the only reason I can see for making `this.sortDirection` public is to use it in the template. 👍🏼

Reply:
I see even a better solution at the moment. What about if we make a `get` for the `sortDirection` and change the property to `private _sortDirection` inside the component. That way we will be sure the property will not be changed from outside. (yea when used in JavaScript everything is possible but you know what I mean xD)